PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -670,6 +676,28 @@
 for modifier in run2_miniAOD_80XLegacy, run2_nanoAOD_94X2016:
   modifier.toReplaceWith(jetSequence, _jetSequence_2016)
 
+#HF shower shape recomputation 
+#Only run if needed (i.e. if default MINIAOD info is missing or outdated because of new JECs...) 
+from RecoJets.JetProducers.hfJetShowerShape_cfi import hfJetShowerShape
+for modifier in run2_miniAOD_80XLegacy, run2_nanoAOD_94X2016, run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94XMiniAODv2, run2_nanoAOD_102Xv1, run2_nanoAOD_106Xv1:
+  modifier.toModify(hfJetShowerShape,jets="updatedJets",vertices="offlineSlimmedPrimaryVertices")
```

Comment:
I was trying to find some other examples of standard reco/miniAOD modules imported and modified without cloning in the NanoAOD setup.
I did not find any (but I may have missed something).
A few lines above `qgtagger` is picked up with a clone.

This no-clone solution will break a possibility to run nano and miniAOD in one step (AFAIK, it is not done in production, but should technically be possible).

@mariadalfonso 
in case I missed other examples, please point me to them, just as a future reference.